scripts/create_broker_secret.py

```diff
@@ -104,17 +124,49 @@ def format_config(config):
         del config['metadata'][key]
     return yaml.dump(config)
 
-def fqname(apb):
-    registries = {'docker.io': 'dh'}
-    registry, org, end = apb.split('/')
 
-    if ":" in end:
-        image, tag = end.split(":")
-    else:
-        image = end
-        tag = 'latest'
+def broker_url():
+    return 'https://' + runcmd('oc get routes --no-headers').split()[1]
 
-    return '-'.join([registries[registry], org, image, tag])
+
+def get_all_apbs():
+    response = requests.get(broker_url() + '/v2/catalog', verify=False)
+    return response.json()['services']
+
+
+
+def fqname(apb):
+    parts = apb.split('/')
```

Comment:
Would doing ```parts = apb.split('/')[-1]``` work here?
It looks like this breaks the image apart like this: ```rthallisey/mediawiki123:latest``` -> ```mediawiki123:latest```

Reply:
no, given either `rthallisey/mediawiki123:latest` or `docker.io/rthallisey/mediawiki123:latest` it should produce `rthallisey-mediawiki123-latest`

Reply:
You could do this:
```python
if apb.count('/') >= 2:
    parts = apb.split("/", 1)[-1]
parts = parts.replace("/", "-").replace(":", "-")
```

Reply:
yeah that's way cleaner